lighthouse-core/report/v2/renderer/crc-details-renderer.js

```diff
@@ -110,7 +110,7 @@ class CriticalRequestChainRenderer {
     const {file, hostname} = Util.parseURL(segment.node.request.url);
     const treevalEl = dom.find('.crc-node__tree-value', chainsEl);
     dom.find('.crc-node__tree-file', treevalEl).textContent = `${file}`;
-    dom.find('.crc-node__tree-hostname', treevalEl).textContent = `(${hostname})`;
+    dom.find('.crc-node__tree-hostname', treevalEl).textContent = hostname ? `(${hostname})` : '';
```

Comment:
nice find :) if it doesn't have a hostname though should we be showing it? what type of request is that, data URI?

Reply:
no it comes back from the parseUrl where a request might be the root so I don't return a hostname.

it actually looks weird so I have to do the origin display inside the report and not in the parseUrl 
![image](https://user-images.githubusercontent.com/1120926/37994241-3df73ef0-3211-11e8-9503-77040106982c.png)

Reply:
Updated to
critical request chain:
![image](https://user-images.githubusercontent.com/1120926/37995870-1ed13efe-3216-11e8-8265-ade92840dae8.png)


preconnect table:
![image](https://user-images.githubusercontent.com/1120926/37995843-0ad44158-3216-11e8-8466-2580a62d9b12.png)